performance_regression_test.py

```diff
@@ -354,8 +355,8 @@ def prepare_mv(self, on_populated=False):
             # Create materialized view
             view_name = base_table_name + '_mv'
             self.log.debug('Create materialized view: {0}.{1}'.format(ks_name, view_name))
-            self.create_materialized_view(ks_name, base_table_name, view_name, ['C0'], ['key'], session,
-                                          mv_columns=['C0', 'key'])
+            create_materialized_view(ks_name, base_table_name, view_name, ['C0'], ['key'], session,
```

Comment:
need to reorder: `create_materialized_view(session, ks_name, base_table_name, view_name, ['C0'], ['key'], mv_columns=['C0', 'key'])`

Reply:
Thanks. updated